docs/docs-content/profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile.md

```diff
@@ -12,6 +12,12 @@ You can update a cluster profile to change the configuration of one or more laye
 
 Cluster profile changes will generate an update notification on all the clusters that are created from the profile. Update notifications include information about all the changes applied to the profile since the initial creation or since the previous update. You can apply cluster updates individually at any time.
 
+:::caution
+
+We do not recommend updating cluster profiles that have been used to deploy clusters. Instead, we recommend creating a new profile version and pushing out the changes to the running clusters. For information on versioning profiles, check out [Version a Cluster Profile](version-cluster-profile.md). 
```

Comment:
@ritawatson @karl-cardenas-coding this one is a little tricky to get right. What we don't recommend is updating a currently deployed cluster profile _version_ in order to push out new changes. Instead, we recommend creating a new version ...  I'll let you decide how to word smith that.

Reply:
@antongisli I see the nuance. Revised.

Reply:
Revised as follows: 

![CleanShot 2023-11-10 at 12 09 54](https://github.com/spectrocloud/librarium/assets/117382432/6e9e67f6-6749-4722-854f-77f8485086e9)